feeadjuster/feeadjuster.py

```diff
@@ -305,4 +346,13 @@ def init(options: dict, configuration: dict, plugin: Plugin, **kwargs):
     "E.g. 0.3 for '70/30'% or 0.6 for '40/60'%.",
     "string"
 )
+plugin.add_option(
+    "feeadjuster-feestrategy",
+    "median",
+    "Sets the per channel fee selection strategy. "
+    "Can be 'median' to use the median fees from peers of peer "
+    "or 'global' to use global config or default values. "
+    "Default: 'median'.",
+    "string"
+)
```

Comment:
Could we use 'global' for default value?

Reply:
Sure we can, wanted your opinion on this approach